neat_internal.h

```diff
@@ -30,6 +30,14 @@
 
 #define NEAT_MAX_NUM_PROTO 4
 
+#if defined(USRSCTP_SUPPORT) || defined(HAVE_NETINET_SCTP_H)
+#define SCTP_SUPPORTED
+#else
+#if defined(__linux__) // Need a further test for Linux SCTP support
+#define SCTP_SUPPORTED
```

Comment:
This is not always true. Just use more specific #ifdefs as in the rest of the code.